sklearn/ensemble/_voting.py

```diff
@@ -252,6 +252,26 @@ def __init__(self, estimators, *, voting='hard', weights=None,
         self.flatten_transform = flatten_transform
         self.verbose = verbose
 
+    def _validate_estimators(self):
+        names, clfs = super()._validate_estimators()
+        # mapping from original labels to encoded ones
+        label_mapping = {c: i for i, c in enumerate(self.classes_)}
+
+        updated_clfs = tuple()
```

Comment:
Hi Thomas, I'm using a tuple to be consistent with the parent class `_BaseHeterogeneousEnsemble`'s `_validate_estimators` method, since that returns two tuples.